src/test/scala/firrtlTests/InfoSpec.scala

```diff
@@ -172,4 +192,51 @@ class InfoSpec extends FirrtlFlatSpec with FirrtlMatchers {
     val expectedInfos = Seq(FileInfo(StringLit("Top.scala 15:14")), FileInfo(StringLit("myfile.fir 6:4")))
     circuitState should containTree { case MultiInfo(`expectedInfos`) => true }
   }
+
+  "source locators for basic register updates" should "be propagated to Verilog" in {
+    val result = compileBody(s"""
+      |input clock : Clock
+      |input reset : UInt<1>
+      |output io : { flip in : UInt<8>, out : UInt<8>}
+      |reg r : UInt<8>, clock
+      |r <= io.in $Info1
+      |io.out <= r $Info2
```

Comment:
This test doesn't make use of `Info2` since it's only testing that the register update is propagated. Would it be clearer to remove it?

Reply:
I think it should be there because if you reversed the `InfoExpr.orElse` logic in `ConstProp` then deleting `Info2` would have the same behavior as this, but leaving `Info2` and reversing that logic would cause this test to fail.

Reply:
With another look I realized that what I said is not what this is checking (I am currently working on tests that do do this so it was on my mind). Will remove `Info2`.